internal/oci/oci_linux.go

```diff
@@ -83,45 +85,47 @@ func newPipe() (parent, child *os.File, err error) {
 	return os.NewFile(uintptr(fds[1]), "parent"), os.NewFile(uintptr(fds[0]), "child"), nil
 }
 
-func loadFactory(root string) (libcontainer.Factory, error) {
-	abs, err := filepath.Abs(root)
+// eventStats gets the stats for the container with the given id from an OCI runtime
```

Comment:
this is a nit and a style difference, but generally I prefer caller functions to be above callee functions, so it can be read from top down.

Reply:
Sure, I moved it one layer down :+1: